src/inet.c

```diff
@@ -21,6 +21,9 @@ static char *id = "$Id: inet.c,v 1.3 2001/12/24 01:09:03 sybalsky Exp $ Copyrigh
 #include <sys/fcntl.h>
 #include <sys/bsdtypes.h>
 #endif /* ISC */
+#ifdef LINUX
```

Comment:
What definition is it picking up from fcntl.h?

Reply:
Ah, you're right, this seems unnecessary. I'll remove it.

Reply:
I think we should (perhaps not in this PR) look at the signal handling.  The _GNU_SOURCE also enables the sighold()/sigrelse() compatibility interfaces, and this should probably migrate forward to the POSIX sigaction/sigprocmask/... API

Reply:
I agree. We should probably have a general todo category for legacy interfaces - we definitely shouldn't be calling `getwd`, for instance (main.c:714; ufs.c:555,578,587; unistd.h:526).